internal/util/aws.go

```diff
@@ -44,14 +44,13 @@ func (u *AWSUtil) UploadDirectoryToS3(localPath string, bucket string, prefix st
 			log.Fatalln("Unable to get relative path:", path, err)
 		}
 		/*
-		#nosec CWE-22: added disclaimer to function docs
-		 */
+			#nosec CWE-22: added disclaimer to function docs
+		*/
 		file, err := os.Open(path)
 		if err != nil {
 			log.Println("Failed opening file", path, err)
 			continue
 		}
-		defer file.Close()
```

Comment:
the linting on these deferred file close calls is kind of annoying. I think in the dce repo I just did `//nolint` comments on them.
I might reconsider whether a failed file close is really a fatal error (line 64). I guess I'm not sure under what conditions that could happen (like if the file disappears?)